doc/static_discovery_configuration_guide.md

```diff
@@ -98,13 +102,32 @@ Also, completing the configuration steps require command line access with root p
 
 The table below gives an overview of different keystore and truststore files that are accessed during the configuration process.
 
-| **Keystore/truststore file** | **Password location** | **Password property** | **Description** |
-|---|---|---|---|
-| `/etc/harmony-ap/ap-keystore.jks` | `/etc/harmony-ap/domibus.properties` | `domibus.security.keystore.password` | Keystore for sign key and certificate. |
-| `/etc/harmony-ap/ap-truststore.jks` | `/etc/harmony-ap/domibus.properties` | `domibus.security.truststore.password` | Truststore for trusted public sign certificates. |
-| `/etc/harmony-ap/tls-keystore.jks` | `/etc/harmony-ap/tomcat-conf/server.xml` | `keystorePass` | Keystore for TLS key and certificate. |
+| **Keystore/truststore file**         | **Password location**                    | **Password property** | **Description** |
+|--------------------------------------|------------------------------------------|-----------------------|----------------- |
+| `/etc/harmony-ap/ap-keystore.jks`    | `/etc/harmony-ap/domibus.properties`     | `domibus.security.keystore.password` | Keystore for sign key and certificate. |
+| `/etc/harmony-ap/ap-truststore.jks`  | `/etc/harmony-ap/domibus.properties`     | `domibus.security.truststore.password` | Truststore for trusted public sign certificates. |
+| `/etc/harmony-ap/tls-keystore.jks`   | `/etc/harmony-ap/tomcat-conf/server.xml` | `keystorePass` | Keystore for TLS key and certificate. |
 | `/etc/harmony-ap/tls-truststore.jks` | `/etc/harmony-ap/tomcat-conf/server.xml` | `truststorePass` | Truststore for trusted public TLS certificates. |
 
+
+#### Note about Using Harmony Access Point Container Version
+
+Harmony Access Point configuration is located at `/var/opt/harmony-ap/etc` instead of `/etc/harmony-ap` (for convinience, `/etc/harmony-ap` is a symbolic link to that directory).
+
+The commands in the guide need to be executed inside the container. For example:
+
+```bash
+sudo keytool -keystore /etc/harmony-ap/ap-keystore.jks -storepass <ap_keystore_password> -list
+```
+
+Can be run inside the container with [`docker exec`](https://docs.docker.com/engine/reference/commandline/exec/) (using "sudo" in the container is unnecessary):
+
+```bash
+docker exec -it <container name> keytool -keystore /etc/harmony-ap/ap-keystore.jks -storepass <ap_keystore_password> -list
```

Comment:
Should it be mentioned that the container uses pkcs12 keystores? Also, the debian version starting from AP version 2.2.0 will use pkcs 12 keystores too. It affects the example commands since the keystore file extension is different (`jks` => `p12`).

Reply:
NEDS-150 changes also the Ubuntu installer to use pkcs12 by default, so the document needs to be updated to reflect that.